app/code/Magento/Catalog/Model/ResourceModel/Category/Tree.php

```diff
@@ -170,16 +191,18 @@ public function getStoreId()
      * @param array $exclude
      * @param boolean $toLoad
      * @param boolean $onlyActive
+     * @param boolean $onlyIncludeInMenu
      * @return $this
      * @SuppressWarnings(PHPMD.CyclomaticComplexity)
      * @SuppressWarnings(PHPMD.NPathComplexity)
      */
-    public function addCollectionData(
```

Comment:
Hi @sodhisarab, since this is an API we need to keep the old method with the old behavior, you can add the new method with the new behavior and use where you need to fix the issue.